README.md

```diff
@@ -23,6 +23,7 @@ Commands:
 
 ```bash
 git clone https://github.com/menkrep1337/XSSCon
+chmod 755 -R XSSCon/
```

Comment:
Yep....

Reply:
At next time, maybe you can merge pull request to master instead of update manually

Reply:
OK